src/backend/backend.ts

```diff
@@ -227,9 +274,6 @@ const messageHandler = (message: Message<any>) => {
         message.content.path,
         message.content.value));
 
-    case MessageType.RouterTree:
```

Comment:
This was never required. Here, the backend responds to messages from the frontend, so this doesn't make sense.